server/utils/validation/addNewFeedbackValidation.ts

```diff
@@ -0,0 +1,15 @@
+import Joi from 'joi';
+
+const addNewFeedbackValidation = (
+  obj: { id: string | number, classId: string, feedback: string },
```

Comment:
It is possible that if you create a new interface type  for feedback that's  will be better